src/data/markdown/docs/30 xk6-browser/01 xk6-browser/01 Browser/version.md

```diff
@@ -0,0 +1,12 @@
+---
+title: 'version()'
+excerpt: 'xk6-browser: Browser.version method'
+---
+
+Returns the browser application's version.
```

Comment:
We need to add a code example here too

Reply:
Resolved in: 938e08855aaf6e2b256cd4a7e192ea9a6aa5f5d5